tools/travis_script_osx

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -ex
+#
+# this script is run on travis for the script stage of mac builds
+#
+
+QMAKE=${QMAKE:-qmake}
+LUPDATE=${LUPDATE:-lupdate}
+LRELEASE=${LRELEASE:-lrelease}
+MACDEPLOYQT=${MACDEPLOYQT:-macdeployqt}
+VERSIONID=${VERSIONID:-$(git rev-parse --short HEAD)}
+
+# build and test the CLI
+$QMAKE GPSBabel.pro && make && make check
+
+# build the GUI
+pushd gui
+$QMAKE app.pro && make
+$LUPDATE app.pro
+$LRELEASE app.pro
+popd
+
+# package the app
+mkdir -p gui/objects/GPSBabelFE.app/Contents/MacOS/translations
+cp gui/*.qm  gui/objects/gpsbabelFE.app/Contents/MacOS/translations
+cp GPSBabel gui/objects/GPSBabelFE.app/Contents/MacOS/gpsbabel
+# this help seems usless, why bother?
+cp -r gui/help/*.html  gui/objects/GPSBabelFE.app/Contents/MacOS
+cp gui/gmapbase.html gui/objects/GPSBabelFE.app/Contents/MacOS
+# what about the COPYING.txt file?
+tools/mac-localize
+
+rm -f gui/objects/GPSBabelFE.dmg
+pushd gui/objects
+$MACDEPLOYQT GPSBabelFE.app -executable=GPSBabelFE.app/Contents/MacOS/gpsbabel -dmg -verbose=2
+popd
+
+# what is in there?
+hdiutil attach -noverify gui/objects/GPSBabelFE.dmg
+find /Volumes/GPSBabelFE -ls
+hdiutil detach /Volumes/GPSBabelFE
+
+mv gui/objects/GPSBabelFE.dmg gui/objects/GPSBabel-${VERSIONID}.dmg
```

Comment:
Are we OK with these version numbers looking like "b9ddf503" and not really sortable in the 'semantic versioning' sense or readily human-recognizable?

Reply:
I added the travis build number, so the dmg is named something like GPSBabel-295-b787e2c.dmg and the appveyor one is GPSBabelTest-1.0.205-b787e2c-Setup.exe where b787e2c is the first 7 characters of the git 40 character revision identifiers (SHA-1 hash).  The hash component ties us to the exact source version in the installer.

Reply:
You can see an example at https://github.com/tsteven4/gpsbabel/releases